docs/source/recipes/howtorestrictblocks.md

```diff
@@ -0,0 +1,48 @@
+---
+myst:
+  html_meta:
+    "description": "How to Restrict blocks per content type or path"
+    "property=og:description": "How to Restrict blocks per content type or path"
+    "property=og:title": "How to Restrict blocks"
+    "keywords": "Volto, Plone, frontend, React, blocks"
+---
+
+# How to Restrict blocks
+
+You can restrict blocks from being added to a content type using the `restrict` key in the configuration object.
+This key can be a boolean or a function.
+If the block is restricted, it won't show in the chooser.
+However, it can be still added programatically or using directly the RESTAPI, so this restriction only applies to the user interface.
```

Comment:
```suggestion
However, it can still be added either programatically or by directly using the REST API, so this restriction only applies to the user interface.
```